service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -1143,4 +1148,6 @@ public boolean isDischargeOfCareApplication() {
             .map(Orders::isDischargeOfCareOrder)
             .orElse(false);
     }
+
+    private List<Element<DocumentWithConfidentialAddress>> documentWithConfidentialAddress;
```

Comment:
maybe call it documentsWithConfidentialAddress as there's more than one potentially

Reply:
sure